kuma/javascript/src/router.jsx

```diff
@@ -514,6 +515,16 @@ export default function Router({
         history.replaceState(initialURL, '', initialURL);
     }
 
+    /**
+     * Reset focus to the window after navigation
+     */
+    function resetFocus() {
+        let documentTitle = document.querySelector('a[class*="logoContainer"]');
```

Comment:
This feels a tiny bit weird, but will be addressed by your follow on PR, so looks good to me.
>  In a follow on PR I am going to reintroduce the skip links for documentation pages. When I do that, I will also move focus to the container for these links as opposed to the logo

Though I do wonder if the variable should be named something other than `documentTitle` if we're going to jump to a container of skip links.

Reply:
This will all go away once https://github.com/mozilla/kuma/pull/5533 lands.